doc/developing.rst

```diff
@@ -126,7 +122,114 @@ Building the HTML documentation
 
 To build the HTML documentation (output in ``doc/build/html``), run::
 
-  $ bin/sphinxbuilder
+  $ sphinx-build doc doc/build/html
+
+Developing Reg, Dectate or Importscan
+-------------------------------------
+
+If you need to adjust the sources of reg, dectate or importscan and test them
+together with morepath, they're available in the src directory.
+There you can edit them and test the changes with morepath, which uses them as
+dependencies when you install Morepath for developement as described above.
+
+Installing tox locally with with different Python versions
+----------------------------------------------------------
+
+If you want to check if Morepath works works with the supported Python versions
```

Comment:
"To be able to easily check whether Morepath works with all Python versions it supports"